phalcon/Config/Factory.zep

```diff
@@ -74,27 +74,27 @@ class Factory extends BaseFactory
             );
         }
 
-        if fetch adapter, config["adapter"] {
-            let className = namespaceClass."\\".camelize(adapter);
-            if !strpos(filePath, ".") {
-                let filePath = filePath.".".lcfirst(adapter);
-            }
+        if !fetch adapter, config["adapter"] {
+            throw new Exception(
+                "You must provide 'adapter' option in factory config parameter."
+            );
+        }
 
-            if className == "Phalcon\\Config\\Adapter\\Ini" {
-                if fetch mode, config["mode"] {
-                    return new {className}(filePath, mode);
-                }
-            } elseif className == "Phalcon\\Config\\Adapter\\Yaml" {
-                if fetch callbacks, config["callbacks"] {
-                    return new {className}(filePath, callbacks);
-                }
-            }
+        let className = namespaceClass . "\\" . camelize(adapter);
+        if !strpos(filePath, ".") {
+            let filePath = filePath . "." . lcfirst(adapter);
+        }
 
-            return new {className}(filePath);
+        if className === \Phalcon\Config\Adapter\Ini::class {
+            if fetch mode, config["mode"] {
+                return new {className}(filePath, mode);
+            }
+        } elseif className === \Phalcon\Config\Adapter\Yaml::class {
```

Comment:
@niden @SidRoberts 
Compile error.
Syntax error in cphalcon/phalcon/Config/Factory.zep on line 92.
I think, this is must be "Phalcon\\Config\\Adapter\\Yaml"

Reply:
I've made https://github.com/phalcon/cphalcon/pull/13951 to fix it. I'm still waiting on AppVeyor but it looks to be passing.